ksuid.go

```diff
@@ -58,6 +57,14 @@ var (
 	Max = KSUID{255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255}
 )
 
+// Must returns ksuid if err is nil and panics otherwise.
+func Must(ksuid KSUID, err error) KSUID {
+	if err != nil {
+		panic(err)
+	}
+	return ksuid
+}
```

Comment:
Adding this function seems unrelated to the modification of `Scan` itself, maybe we can split it in a separate PR?